src/pages/Home.jsx

```diff
@@ -1,42 +1,69 @@
 import { useDispatch, useSelector } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import { useState } from 'react';
 import Form from '../components/Form';
+import 'react-circular-progressbar/dist/styles.css';
 import './home.css';
 import { deleteBook, deleteAPIBook } from '../components/redux/books/booksSlice';
 
 // Import the initialState and display
 export default function Home() {
   const dispatch = useDispatch();
   const books = useSelector((state) => state.book.books);
+  const [percentage, newPercentage] = useState(65);
 
+  const handlePercentage = (e) => {
+    e.preventDefault();
+    newPercentage(Math.floor(Math.random() * 100));
+  };
   return (
     <div className="bookList">
       { (books ?? []).filter((b) => b.item_id).map((book) => (
         <div className="eachBook" key={book.item_id} id={book.item_id}>
           <div className="left-pan">
             <h4>{book.category}</h4>
             {' '}
-            <h1>{book.author}</h1>
+            <h1>{book.title}</h1>
             {' '}
-            <p className="booktitle">{book.title}</p>
+            <p className="booktitle">{book.author}</p>
             {' '}
             <div className="buttonDiv">
               {' '}
               <button type="button">Comments</button>
-              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>REMOVE</button>
+              <button className="remove" type="button" onClick={() => { dispatch(deleteBook(book.item_id)); dispatch(deleteAPIBook(book.item_id)); }}>Remove</button>
               <button type="button">Edit</button>
             </div>
           </div>
           <div className="middle-pan">
-            <div className="circle" />
+            <div className="circle">
+              <CircularProgressbar
+                value={percentage}
+                strokeWidth={8}
+                styles={{
+                  path: {
+                    stroke: '#3f51b5',
+                    strokeLinecap: 'butt',
+                    transition: 'stroke-dashoffset 0.5s ease 0s',
+                  },
+                  trail: {
+                    stroke: '#d6d6d6',
+                    strokeLinecap: 'butt',
+                  },
+                }}
+              />
+            </div>
```

Comment:
As it's a project requirement to use the same design and styles as used in the ZEPLIN template so please consider using the same set of styles. Currently, there you are using different colors please consider using the same color as in Zeplin's design.

## YOUR DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/230605265-8872ff2f-767c-4344-99fb-08bd92c99666.png)


## ZEPLIN DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/230605285-4400d977-d85e-4eb9-8635-3c216398a481.png)